src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,7 +1,13 @@
 package mate.academy
 
 fun greetUsers() {
-    // implement function
+    do {
+        println("Please enter a name (or press Enter to quit):")
+        val name = readlnOrNull()
+        if (name != "") {
```

Comment:
Consider using `if (!name.isNullOrEmpty())` to handle cases where `name` might be `null`. This will prevent any potential issues when `readlnOrNull()` returns `null`.